sdk/keyvault/azure-security-keyvault-keys/src/key_rotation_policy.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/internal/json/json.hpp>
+#include <azure/core/internal/json/json_optional.hpp>
+
+#include "azure/keyvault/keys/key_client_models.hpp"
+#include "private/key_constants.hpp"
+#include "private/key_serializers.hpp"
+
+using namespace Azure::Security::KeyVault::Keys;
+using namespace Azure::Core::Json::_internal;
+using Azure::Core::_internal::PosixTimeConverter;
+
+KeyRotationPolicy _detail::KeyRotationPolicySerializer::KeyRotationPolicyDeserialize(
+    Azure::Core::Http::RawResponse const& rawResponse)
+{
+  auto body = rawResponse.GetBody();
+  return KeyRotationPolicyDeserialize(body);
+}
+
+KeyRotationPolicy _detail::KeyRotationPolicySerializer::KeyRotationPolicyDeserialize(
+    std::vector<uint8_t> const& body)
+{
+  auto jsonParser = Azure::Core::Json::_internal::json::parse(body);
```

Comment:
nit.  Use `const` whenever you are not expecting to mutate the value.

```suggestion
  auto const jsonParser = Azure::Core::Json::_internal::json::parse(body);
```

Reply:
done